src/constants.py

```diff
@@ -6,4 +6,5 @@
 MIN_MASS = SCALE_FACTOR**3
 
 G = gravitational_constant * 1e7
+LY = light_year
```

Comment:
I needed it when I was trying to implement skipping distant objects' gravity calculation. Didn't work though.